.github/workflows/verify.yml

```diff
@@ -206,7 +206,7 @@ jobs:
     needs: prepare-test-cache
     strategy:
       matrix:
-        python-version: [3.9, "3.10"]
```

Comment:
Is the library removing 3.9 support?

Reply:
Not intended as such

Reply:
Just wanted to move default to 3.10 and still support 3.9 - as it's mainly focussed at HA we wouldn't drop until Core drops support

Reply:
Its a matrix, there is no default :)